google/cloud/forseti/enforcer/enforcer.py

```diff
@@ -161,7 +115,54 @@ def main(argv):
 
     del argv
```

Comment:
Does this delete statement affect the values argparser is going to get from the user?

Reply:
It's unused and was from the begining.

Reply:
I switched away from apputils here and argv isn't a problem anymore or at least we ignore it because we don't use it now.